website/docs/tasks/originate.md

```diff
@@ -0,0 +1,61 @@
+---
+title: originate
+---
+
+## Overview
+
+Taq `originate` is used to originate one or many Michelson smart contracts to one or more sandboxes or networks. Before originating a contract, you must define the initial storage value for the contract in the envrionment you are originating to. Instructions for this can be found [here](/docs/config/environments)
+
+[Taqueria Environments](/docs/config/environments) are used to specify the target for origination
+
+The task expects the Michelson files to be in the `/artifacts` directory
+
+:::note
+Contracts must be added to the [Contract Registry](/docs/tasks/add-contract) (`taq add-contract example.mligo`) in order to be originated
+:::
+
+## Plugin Implementations
+
+This task is implemented by the following plugins:
+
+| Plugin Name                  | Description                                    |
+| ---------------------------- | ---------------------------------------------- |
+| `@taqueria/plugin-taquito`   | Originates contracts to a target environment   |
+
+### Command
+
+```shell
+taq originate [path]
+``` 
+
+
+### Task Details
+
+| Task Name      | Command                       | Type                | Description                                             | 
+| -------------- | ----------------------------- | ------------------- | ------------------------------------------------------- |
+| `originate`    | `taq originate [path]`        | Plugin - Taquito    | Originates one, multiple, or all contracts in a project |
```

Comment:
I don't think there is a way to originate multiple contracts, it's either single or all